samples/quadratic-assignment-problem/quadratic-assignment-problem.ipynb

```diff
@@ -316,7 +316,7 @@
    "source": [
     "## Submitting problem to the Azure Quantum solver\n",
     "\n",
-    "The SlcTerm functionality is only available on the [Population Annealing](https://docs.microsoft.com/azure/quantum/optimization-population-annealing#parameterized-population-annealing) and [Substochastic Monte Carlo](https://docs.microsoft.com/azure/quantum/optimization-substochastic-monte-carlo#parameterized-substochastic-monte-carlo) solvers. For this sample we will use the population annealing solver with its default parameters (since the problem is trivial).\n",
+    "The SlcTerm functionality is available for all Microsoft QIO CPU solvers, you can find a full list in [our documentation](https://docs.microsoft.com/en-us/azure/quantum/optimization-slc-term#availability). For this sample we will use the population annealing solver with its default parameters (since the problem is trivial).\n",
```

Comment:
Need to remove localisation from URLs
```suggestion
    "The SlcTerm functionality is available for all Microsoft QIO CPU solvers, you can find a full list in [our documentation](https://docs.microsoft.com/azure/quantum/optimization-slc-term#availability). For this sample we will use the population annealing solver with its default parameters (since the problem is trivial).\n",
```